src/main/java/ch/njol/skript/doc/HTMLGenerator.java

```diff
@@ -603,6 +617,31 @@ private String generateEvent(String descTemp, SkriptEventInfo<?> info, @Nullable
 		String[] keywords = info.getKeywords();
 		desc = desc.replace("${element.keywords}", keywords == null ? "" : Joiner.on(", ").join(keywords));
 
+		// Event Values
+		Set<String> eventValuesSet = new TreeSet<>(); // sort and remove duplicates due to having multiple events
+		for (Class<? extends Event> event : info.events) {
+			for (Class<? extends Event> index : eventValuesCache.keySet()) {
+				if (!(event.isAssignableFrom(index) || index.isAssignableFrom(event))) // index.isAssignableFrom(event) has inaccuracy like TeleportCause in PlayerMoveEvent but also needed for superclass event values such as event-location
+					continue;
+				for (EventValues.EventValueInfo<?, ?> eventValueInfo : eventValuesCache.get(index)) {
+					if (eventValueInfo.isEventExcluded(event))
+						continue;
+					String prefix = "";
+					if (eventValueInfo.getTime() == EventValues.TIME_PAST)
+						prefix = "past ";
+					else if (eventValueInfo.getTime() == EventValues.TIME_FUTURE)
+						prefix = "future ";
+					String className = eventValueInfo.c.getSimpleName().toLowerCase(Locale.ENGLISH);
+					eventValuesSet.add(prefix + "event-" + (className.contains("[]") ? Utils.toEnglishPlural((className.replace("[]", ""))) : className));
+				}
+			}
+		}
+		List<String> eventValues = new ArrayList<>(0);
```

Comment:
```suggestion
		List<String> eventValues = new ArrayList<>();
```
No need to initialize the array list here, unless you want to do eventValuesSet.size()